<launch>
  <node pkg="dich" type="replay" name="replay" required="true">
    <!-- <param name="framerate" value="20" /> -->
    <param name="path"                value="teach.mpg" />
    <param name="path_replay_offline" value="replay.mpg" />
    <param name="headless" value="true" />
    <param name="queue" value="0" />
  </node>
</launch>
